lib/translator/report_luns.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_LUN_TRANSLATOR_H
+#define LIB_LUN_TRANSLATOR_H
+
+#include "absl/types/span.h"
+
+#include "common.h"
+#include "lib/scsi_defs.h"
+#include "third_party/spdk_defs/nvme_defs.h"
+
+namespace translator {
+
+StatusCode ReportLunsToNvme(absl::Span<const uint8_t> scsi_cmd,
+                            nvme_defs::GenericQueueEntryCmd& nvme_cmd);
+
+StatusCode ReportLunsToScsi(nvme_defs::GenericQueueEntryCmd,
```

Comment:
The definition of this takes a const reference, but this prototype takes a value. That mismatch will lead to linker errors.

Reply:
Thanks!